ui/src/app/app-routing.module.ts

```diff
@@ -39,8 +39,9 @@ const routes: Routes = [
   {
     path: 'dashboard',
     component: DashboardComponent,
-    //TODO: (zach) dashboard need a query param of project id before it can be activated
+    //TODO: (zach) dashboard need query params of projectId also timeFrame before it can be activated
```

Comment:
Put the action item in the todo, what needs to be fixed? Is it that if you route to /dashboard without project you aren't properly redirected to the projects page?

Reply:
Yes. If the projectId is missing, it shows 400 error. Adding this action to the TODO.